examples/metalk8s-solution-example/manifest.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+
+"""Utility script to generate a Solution manifest from given arguments."""
+
+import argparse
+import sys
+import yaml
+
+
+SOLUTION_MANIFEST = {
+    'apiVersion': 'solutions.metalk8s.scality.com/v1alpha1',
+    'kind': 'Solution',
+    'metadata': {
+        'name': '',
+        'annotations': {},
+        'labels': {},
+    },
+    'spec': {
```

Comment:
The list of extra images could be added as an option to the script, so we can check that overriding the default behaviour works as expected (we could build an ISO with a wrong manifest and check how our tooling behaves). But not needed for now, so feel free to leave this as debt (with a ticket to track and test) :)

Reply:
Added the list, so we just have to add the test now.
I'll create a ticket.

Reply:
ping

Reply:
#2618